stories/Button.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import centered from "@storybook/addon-centered/react";
+import Button from "../components/base/Button";
+
+storiesOf("Button", module)
```

Comment:
Let's move this file to the `stories/base` directory and change this line to `storiesOf("base|Button", module) `

Reply:
Sounds good! I'll get those changes in.